runtime/src/main/cpp/Memory.cpp

```diff
@@ -2600,6 +2600,10 @@ void freezeSubgraph(ObjHeader* root) {
 
   MEMORY_LOG("Freeze subgraph of %p\n", root)
 
+  if (root->type_info() == theWorkerBoundReferenceTypeInfo) {
+    FreezeWorkerBoundReference(root);
```

Comment:
Does this work for non-root `WorkerBoundReference`, found in subgraph to be frozen?

Reply:
Missed it, thanks!